html/index.html

```diff
@@ -0,0 +1,480 @@
+    
+            </div>
+    
+    
+    
+        </div>
+    
+        <div class="second-container">
+    
+            <div class="inner-second-container">
+    
+                <div class="top-image">
+    
+                    <img src="../assets/Snapshoot Portfolio 3.png" alt="">
+    
+                </div>
+    
+                <div class="text-inner-container">
+    
+                    <h2> Tonic </h2>
+    
+                    <div class="canopy-inner-container">
+    
+                        <p id="text1">CANOPY</p>
+    
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+    
+                    </div>
+    
+                    <div class="canopy-daily">
+    
+                        <p>
+                            A daily selection of privately 
+                            personalized reads; no accounts 
+                            or sign-ups required.
+        
+                        </p>
+    
+                    </div>
+    
+    
+                    <div class="canopy-list">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+    
+                </div>
+    
+                <div >
+                    <button id="second-container-button">See Project</button>
+                </div>
+    
+            </div>
+    
+    
+    
+        </div>
+    
+        <div class="second-containerl">
+    
+            <div class="inner-second-containerl">
+    
+                <div class="top-image">
+    
+                    <img src="../assets/Snapshoot Portfolio 4.png" alt="">
+    
+                </div>
+    
+                <div class="text-inner-container">
+    
+                    <h2> Multi-Post <br> Stories </h2>
+    
+                    <div class="canopy-inner-container">
+    
+                        <p id="text1">CANOPY</p>
+    
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+    
+                    </div>
+    
+                    <div class="canopy-daily">
+    
+                        <p>
+                            A daily selection of privately 
+                            personalized reads; no accounts 
+                            or sign-ups required.
+        
+                        </p>
+    
+                    </div>
+    
+    
+                    <div class="canopy-list">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+    
+                </div>
+    
+                <div >
+                    <button id="second-container-button">See Project</button>
+                </div>
+    
+            </div>
+    
+        </div>
+    
+        <div class="third-container">
+    
```

Comment:
- [ ] As I mentioned above, this must be a `section` tag too, consider changing it to as per the requirement